storefront/theme.json

```diff
@@ -15,5 +18,161 @@
     "color-grey-lightest": "#ececec",
     "color-danger": "#dc3545",
     "border-radius": "0"
+  },
+  "localizedRoutes": {
+    "privacy": {
```

Comment:
Instead of slug:privacy_slug, the key of the object could be the slug key already? And what do the localizations do here when there is a slugs de.js?

Reply:
@pozylon they are used for rewrites. used with `generateRouteRewites`